rpcs3/Emu/Cell/Modules/cellAudio.cpp

```diff
@@ -1145,7 +1145,6 @@ error_code cellAudioQuit()
 	}
 
 	// TODO
-	g_audio.keys.clear();
```

Comment:
Please add a comment to not clear it for the next person who thinks it's missing.

Reply:
Also, just wondering, shouldn't the queue still be deleted in the removequeue function? I think it's missing

Reply:
Nope, I was fool, you were right.